Formula/postgresql@95.rb

```diff
@@ -0,0 +1,147 @@
+class PostgresqlAT95 < Formula
+  desc "Object-relational database system"
+  homepage "https://www.postgresql.org/"
+  url "https://ftp.postgresql.org/pub/source/v9.5.5/postgresql-9.5.5.tar.bz2"
```

Comment:
I was [planning on updating to 9.5.6](https://github.com/Homebrew/homebrew-versions/pull/1533) when I saw that homebrew-versions is deprecated. I was going move postgresql95 here first, then update after. What do you think, @MikeMcQuaid?